src-tauri/Cargo.toml

```diff
@@ -38,6 +38,13 @@ windows = { version = "0.52.0", features = [
     "Win32_System",
     "Win32_System_Threading",
 ] }
+errno = "0.2.4"
+libc = "0.2.43"
+ctrlc = "3.1.1"
+fs2 = "0.4.3"
```

Comment:
Unused dependencies: `libc`, `errno` & `fs2`. The more dependencies, the more binary size and compile time. Keep our list dependencies as clean as possible.